google/cloud/pubsub_v1/subscriber/message.py

```diff
@@ -406,6 +439,8 @@ def modify_ack_deadline_with_response(self, seconds: int) -> "futures.Future":
             will be thrown.
 
         """
+        if self.opentelemetry_data:
+            self.opentelemetry_data.add_subscribe_span_event("modack start")
```

Comment:
Same as above: make this event when the RPC is called.

Reply:
Done. Response: https://github.com/googleapis/python-pubsub/pull/1252#discussion_r1769706317